tests/test_p2p_daemon_bindings.py

```diff
@@ -94,7 +94,7 @@ async def test_write_unsigned_varint_overflow(integer):
         await write_unsigned_varint(s, integer)
 
 
-@pytest.mark.parametrize("integer", (-1, -(2 ** 32), -(2 ** 64), -(2 ** 128)))
+@pytest.mark.parametrize("integer", (-1, -(2**32), -(2**64), -(2**128)))
```

Comment:
```suggestion
@pytest.mark.parametrize("integer", (-1, -2**32, -2**64, -2**128))
```

Reply:
Disagree
Parenthesis save a few seconds of understanding in cases like this

Reply:
I'm also for keeping the brackets in this case